auth-api/src/auth_api/services/entity.py

```diff
@@ -133,3 +133,13 @@ def get_contact_for_business(business_identifier):
         if contact_link is None:
             return None
         return contact_link.contact
+
+    @staticmethod
+    def validate_pass_code(business_identifier, pass_code):
+        """Get the contact for a business identified by the given id."""
+        entity = EntityModel.find_by_business_identifier(business_identifier)
+        if entity is None:
+            return None
```

Comment:
Should this case return `False` instead of `None`?

Reply:
I thought None will kind of give more information, can identify if it's pass_code error or entity number error. The program call this service have chance to deal with this again to give more detailed error message.

Reply:
Understood, but that creates a lot of coupling between the caller and your code.  The caller has to know what a return value of None means in this case.  It's also not a good idea to have an ambiguous return type for your functions (in this case it's a hybrid Boolean/None).

Reply:
> Understood, but that creates a lot of coupling between the caller and your code. The caller has to know what a return value of None means in this case. It's also not a good idea to have an ambiguous return type for your functions (in this case it's a hybrid Boolean/None).

Totally agree, return type should always be same

Reply:
updated.